### PR TITLE
feat: more german translation

### PR DESCRIPTION
**Pull request by NickUfer**
This should add all relevant german translation.

### CODE REVIEW

File: src/main/resources/translations/german.json [hunk 1]
```diff
@@ -2,9 +2,102 @@
...
+  "SAVE_AS_DEX": "Speichern als DEX...",
+  "SAVE_AS_APK": "Speichern als APK...",
+  "DECOMPILE_SAVE_OPENED_CLASSES": "Alle geöffneten Klassen dekompilieren & speichern",
+  "DECOMPILE_SAVE_ALL_CLASSES": "Alle Klassen dekompilieren & speichern",
+  "RECENT_FILES": "Letzte geöffnete Dateien",
+  "ABOUT": "Über Bytecode Viewer",
+  "EXIT": "Bytecode Viewer beenden",
+
+  "VIEW": "Ansicht",
+  "VISUAL_SETTINGS": "Visuelle Einstellungen",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:19]:
```suggestion
  "VISUAL_SETTINGS": "Grafikeinstellungen",
```

File: src/main/resources/translations/german.json [hunk 2]
```diff
@@ -2,9 +2,102 @@
...
+  "LANGUAGE": "Sprache",
+
+  "TODO1": "Diese existieren nur als Enums",
+  "WINDOW_THEME": "Fenster Erscheinungsbild",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:23]:
```suggestion
  "WINDOW_THEME": "Fenster-Erscheinungsbild",
```

File: src/main/resources/translations/german.json [hunk 3]
```diff
@@ -2,9 +2,102 @@
...
+  "SYSTEM_THEME": "Wie Betriebssystem",
+  "DARK_THEME": "Dark-Mode",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:25]:
```suggestion
  "DARK_THEME": "Dunkler Modus",
```

File: src/main/resources/translations/german.json [hunk 4]
```diff
@@ -2,9 +2,102 @@
...
+  "LIGHT_THEME": "Light-Mode",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:26]:
```suggestion
  "LIGHT_THEME": "Heller Modus",
```

File: src/main/resources/translations/german.json [hunk 5]
```diff
@@ -2,9 +2,102 @@
...
+  "TEXT_AREA_THEME": "Textbereich Erscheinungsbild",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:27]:
```suggestion
  "TEXT_AREA_THEME": "Textbereich-Erscheinungsbild",
```

File: src/main/resources/translations/german.json [hunk 6]
```diff
@@ -2,9 +2,102 @@
...
+  "DEFAULT_RECOMMENDED_LIGHT": "Hell (Empfohlener Light-Mode)",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:28]:
```suggestion
  "DEFAULT_RECOMMENDED_LIGHT": "Hell (Empfohlen bei hellem Modus)",
```

File: src/main/resources/translations/german.json [hunk 7]
```diff
@@ -2,9 +2,102 @@
...
+  "DARK_RECOMMENDED_DARK": "Dunkel (Empfohlener Dark-Mode)",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:29]:
```suggestion
  "DARK_RECOMMENDED_DARK": "Dunkel (Empfohlen bei dunklem Modus)",
```

File: src/main/resources/translations/german.json [hunk 8]
```diff
@@ -2,9 +2,102 @@
...
+  "DEFAULT_ALT": "Standard Alternativ",
+  "ECLIPSE": "Eclipse",
+  "INTELLIJ": "Intellij",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:32]:
```suggestion
  "INTELLIJ": "IntelliJ",
```

File: src/main/resources/translations/german.json [hunk 9]
```diff
@@ -2,9 +2,102 @@
...
+  "VISUAL_STUDIO": "Visual Studio",
+  "DRUID_DARK": "Druid (Dark-Mode)",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:34]:
```suggestion
  "DRUID_DARK": "Druid (Dunkler Modus)",
```

File: src/main/resources/translations/german.json [hunk 10]
```diff
@@ -2,9 +2,102 @@
...
+  "MONOKAI_DARK": "Monokai (Dark-Mode)",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:35]:
```suggestion
  "MONOKAI_DARK": "Monokai (Dunkler Modus)",
```

File: src/main/resources/translations/german.json [hunk 11]
```diff
@@ -2,9 +2,102 @@
...
+  "PANE_1": "Fenster 1",
+  "PANE_2": "Fenster 2",
+  "PANE_3": "Fenster 3",
+  "NONE": "Keins",
+  "EDITABLE": "Editierbar",
+
+  "SETTINGS": "Einstellungen",
+  "COMPILE_ON_SAVE": "Bei Speichern kompilieren",
+  "COMPILE_ON_REFRESH": "Bei Aktualisierung kompilieren",
+  "REFRESH_ON_VIEW_CHANGE": "Aktualisieren bei Änderung der Ansicht",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:53]:
```suggestion
  "REFRESH_ON_VIEW_CHANGE": "Bei Änderung der Ansicht aktualisieren",
```

File: src/main/resources/translations/german.json [hunk 12]
```diff
@@ -2,9 +2,102 @@
...
+  "DECODE_APK_RESOURCES": "APK Ressourcen dekodieren",
+  "APK_CONVERSION": "APK Umwandlung",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:55]:
```suggestion
  "APK_CONVERSION": "APK-Umwandlung",
```

File: src/main/resources/translations/german.json [hunk 13]
```diff
@@ -2,9 +2,102 @@
...
+  "DEX_TO_JAR": "Dex2Jar",
+  "ENJARIFY": "Enjarify",
+  "UPDATE_CHECK": "Nach Updates suchen",
+  "DELETE_UNKNOWN_LIBS": "Unbekannte/Veraltete Bibliotheken löschen",
+  "FORCE_PURE_ASCII_AS_TEXT": "Ascii als Zeichenkodierung erzwingen",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:60]:
```suggestion
  "FORCE_PURE_ASCII_AS_TEXT": "ASCII als Zeichenkodierung erzwingen",
```

File: src/main/resources/translations/german.json [hunk 14]
```diff
@@ -2,9 +2,102 @@
...
+  "SET_PYTHON_27_EXECUTABLE": "Ausführbare Python 2.7 Datei festlegen",
+  "SET_PYTHON_30_EXECUTABLE": "Ausführbare Python 3.X Datei festlegen",
+  "SET_JRE_RT_LIBRARY": "JRE RT Bibliothek festlegen",
+  "SET_OPTIONAL_LIBRARY_FOLDER": "Optionalen Bibliothekenordner festlegen",
+  "SET_JAVAC_EXECUTABLE": "Ausführbare Javac Datei festlegen",
+
+  "TODO3": "Diese existieren nur als Key/Value-Paare",
+  "TODO4": "Ende",
+
+  "BYTECODE_DECOMPILER": "Bytecode Dekompilierer",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:70]:
```suggestion
  "BYTECODE_DECOMPILER": "Bytecode-Dekompilierer",
```

File: src/main/resources/translations/german.json [hunk 15]
```diff
@@ -2,9 +2,102 @@
...
+  "DEBUG_HELPERS": "Debug Helfer",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:71]:
```suggestion
  "DEBUG_HELPERS": "Debug-Helfer",
```

File: src/main/resources/translations/german.json [hunk 16]
```diff
@@ -2,9 +2,102 @@
...
+  "SHOW_MAIN_METHODS": "Main-Methoden anzeigen",
+  "SHOW_ALL_STRINGS": "Alle Strings anzeigen",
+  "REPLACE_STRINGS": "Strings ersetzen",
+  "STACK_FRAMES_REMOVER": "Stack-Frames-Entferner ausführen",
+  "ZKM_STRING_DECRYPTER": "ZKM-String-Decrypter ausführen",
+  "ALLATORI_STRING_DECRYPTER": "Allatori-String-Decrypter ausführen",
+  "ZSTRINGARRAY_DECRYPTER": "ZStringArray-Decrypter ausführen",
+
+  "TODO5": "Diese existieren nur als Key/Value-Paare",
+  "PROCYON_DECOMPILER": "Procyon Dekompilierer",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:88]:
```suggestion
  "PROCYON_DECOMPILER": "Procyon-Dekompilierer",
```

File: src/main/resources/translations/german.json [hunk 17]
```diff
@@ -2,9 +2,102 @@
...
+  "CRF_DECOMPILER": "CFR Dekompilierer",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:89]:
```suggestion
  "CRF_DECOMPILER": "CFR-Dekompilierer",
```

File: src/main/resources/translations/german.json [hunk 18]
```diff
@@ -2,9 +2,102 @@
...
+  "FERNFLOWER_DECOMPILER": "FernFlower Dekompilierer",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:90]:
```suggestion
  "FERNFLOWER_DECOMPILER": "FernFlower-Dekompilierer",
```

File: src/main/resources/translations/german.json [hunk 19]
```diff
@@ -2,9 +2,102 @@
...
+  "JADX_DECOMPILER": "JADX Dekompilierer",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:91]:
```suggestion
  "JADX_DECOMPILER": "JADX-Dekompilierer",
```

File: src/main/resources/translations/german.json [hunk 20]
```diff
@@ -2,9 +2,102 @@
...
+  "JD_DECOMPILER": "JD-GUI Dekompilierer",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:92]:
```suggestion
  "JD_DECOMPILER": "JD-GUI-Dekompilierer",
```

File: src/main/resources/translations/german.json [hunk 21]
```diff
@@ -2,9 +2,102 @@
...
+  "BYTECODE_DISASSEMBLER": "Bytecode Disassembler",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:93]:
```suggestion
  "BYTECODE_DISASSEMBLER": "Bytecode-Disassembler",
```

File: src/main/resources/translations/german.json [hunk 22]
```diff
@@ -2,9 +2,102 @@
...
+
+  "FILES": "Dateien",
+  "QUICK_FILE_SEARCH_NO_FILE_EXTENSION": "Dateien Schnellsuche öffnen (ohne Dateiendungen)",
```
Comment (ThexXTURBOXx) [inline src/main/resources/translations/german.json:96]:
```suggestion
  "QUICK_FILE_SEARCH_NO_FILE_EXTENSION": "Dateien-Schnellsuche öffnen (ohne Dateiendungen)",
```

**Review by ThexXTURBOXx — CHANGES_REQUESTED** [review]
Looks very good. Thank you for your work! :)

I would do some small adjustments and then I am completely satisfied! :D

**Comment by NickUfer** [comment]
Done :)

**Comment by ThexXTURBOXx** [comment]
Thank you very much. I approve this PR :)

**Review by ThexXTURBOXx — APPROVED** [review]

**Comment by Konloch** [comment]
Great work, thank you!

**Pull request merged**